canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -105,56 +110,45 @@ $toolbar-divider-width: 1px;
 	width: 0;
 	display: inline-block;
 }
-
-
 .toolbar-overflow-item {
 	background-color: $ui-01;
 	display: inline-block;
-
 	& button {
 		padding: 0;
 		height: $toolbar-button-height;
 		min-height: 30px;
 		background-color: $ui-01;
 		border-right: $toolbar-divider-width solid $ui-03;
 	}
-
 	& button:hover {
 		background-color: $hover-field;
 	}
-
 	& button:disabled:hover {
 		background-color: $ui-01;
 	}
 }
-
 .toolbar-overflow-menu-item button {
 	width: 100%;
 	padding-top: 2px;
 	padding-bottom: 2px;
 	min-height: 20px;
 }
-
```

Comment:
Can you undo these changes? Its more clear to have the newlines while reading this file

Reply:
The reason to chose these unique classes is, to separate my style from the icons within the toolbar in canvas and other toolbar icons in table buttons since they both share pretty much the same classes except the ones I chose.

```
.togglePalette-action,
	.undo-action,.redo-action,
	.cut-action,.copy-action,
	.paste-action,
	.createAutoComment-action,
	.deleteSelectedObjects-action{
		padding: 3px;
	}
```
Looks like these extra new line has not been fixed...I'll see what else I can do for it

Reply:
> <img alt="image" width="406" src="https://user-images.githubusercontent.com/1628100/197057936-a06ea70c-741d-4785-9538-12e4750903bf.png">
> 
The reason to chose these unique classes is, to separate my style from the icons within the toolbar in canvas and other toolbar icons in table buttons since they both share pretty much the same classes except the ones I chose.

.togglePalette-action,
	.undo-action,.redo-action,
	.cut-action,.copy-action,
	.paste-action,
	.createAutoComment-action,
	.deleteSelectedObjects-action{
		padding: 3px;
	}
Looks like these extra new line has not been fixed...I'll see what else I can do for it

Reply:
>The reason to chose these unique classes is, to separate my style from the icons within the toolbar in canvas and other toolbar icons in table buttons

You can add additional class name to differentiate when toolbar is inside a table or not

Reply:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/89609176/197632469-3f6bf936-e13e-42b7-a9e4-5b7ed1de2fd2.png">

Reply:
This will change everything. You can create a new class name in the code specifically for toolbar in tables, that way the styles will only apply to that instead of the canvas toolbar too